### ISSUE TITLE
hosts file editor - description of settings is not explanatory

### ISSUE THREAD

**Issue by runmode** [MicrosoftDocs/windows-uwp#4909]
### Type of issue

Missing information

### Feedback

The description of the following settings just repeats the wording of the setting and gives no useful information:

"**Additional lines position**: Default value is Top. If Bottom is selected, the **file header** is moved below hosts settings to the bottom." This is **contradictory**. Are additional entries added at the bottom, or is teh file header moved to to bottom? And most important: **What is the consequence of this choice?**

"**Consider loopback addresses as duplicates**: Loopback addresses (like 127.0.0.1 and ::1) are considered as duplicates." This just explains what a loopback address is, but what is the result of this setting (complete phrase that will be added) and **what is the consequence of "considering" or "not considering"?** Will it be ignored or dropped or interpreted twice or only the first occurrance interpreted, or the last occurrance, or whatever? Which choice is recommended for which purpose?

Btw, the movies are running so fast, hard to follow.

### Page URL

https://learn.microsoft.com/en-us/windows/powertoys/hosts-file-editor

### Content source URL

https://github.com/MicrosoftDocs/windows-dev-docs/blob/docs/hub/powertoys/hosts-file-editor.md

### Author

@alvinashcraft

### Document Id

bea3fb1d-1434-53bc-3712-d62ee7bb2beb

**Comment by Jay-o-Way** [comment]
Yes, good feedback. @taras-janea Can you pick this up please.
